sql/expression/literal.go

```diff
@@ -64,24 +65,28 @@ func (lit *Literal) Eval(ctx *sql.Context, row sql.Row) (interface{}, error) {
 }
 
 func (lit *Literal) String() string {
```

Comment:
This might have been an easy one unit test, list of tests that create a `lit := Literal(input)` from an input string, and then compare output of `lit.String()` to an expected value. Not that the other tests aren't also helpful.

Reply:
I'll put a pin in that. Not going to add due to other priorities and going to ship what we got